configuration/config-txt/memory.md

```diff
@@ -30,6 +30,14 @@ The `gpu_mem_512` command sets the GPU memory in megabytes for Raspberry Pis wit
 
 The `gpu_mem_1024` command sets the GPU memory in megabytes for Raspberry Pis with 1GB or more of memory. (It is ignored if memory size is smaller than 1GB). This overrides `gpu_mem`.
 
+## total_mem
```

Comment:
@JamesH65 Are there any restrictions on the values `total_mem` can take? Does it work on all Pis?

Reply:
min 128 max memory on board. Will adjust this PR to make that clear.